src/aap_eda/core/models/source.py

```diff
@@ -0,0 +1,50 @@
+#  Copyright 2022 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import uuid
+
+from django.db import models
+
+from aap_eda.core.enums import RestartPolicy
+
+__all__ = "Source"
+
+
+class Source(models.Model):
+    class Meta:
+        db_table = "core_source"
+        indexes = [
+            models.Index(fields=["uuid"], name="ix_source_uuid"),
+            models.Index(fields=["name"], name="ix_source_name"),
+        ]
+
+    uuid = models.UUIDField(primary_key=True, default=uuid.uuid4)
```

Comment:
@hsong-rh Is the default computed every time or just once?

Reply:
just once, as primary key it is executed at the creation of the record.